src/Microsoft.Health.Fhir.Core.UnitTests/Features/Definition/SearchParameterDefinitionBuilderTests.cs

```diff
@@ -31,9 +32,7 @@ public SearchParameterDefinitionBuilderTests()
         }
 
         [Theory]
-        [InlineData("SearchParametersWithNullEntry", "FHIR Serialization requires an element to have non-null data (at path 'entry[0]')")]
-        [InlineData("SearchParametersWithInvalidType", "Literal '' is not a valid value for enumeration 'SearchParamType'")]
-        [InlineData("SearchParametersWithInvalidBase", "Literal '' is not a valid value for enumeration 'ResourceType'")]
```

Comment:
@jackliums These seem to have changed with in the serializers new permissiveParsing setting